package.json

```diff
@@ -17,8 +17,8 @@
   },
   "dependencies": {
     "@nuxtjs/style-resources": "^1.0.0",
-    "@spryker-vsf/api": "^0.3.0",
-    "@spryker-vsf/composables": "^0.2.0",
+    "@spryker-vsf/api": "^0.4.0",
+    "@spryker-vsf/composables": "^0.3.0",
```

Comment:
Please update to next version first to check if it's working properly.

Reply:
Done